cypress/fixtures/suite/backoffice/product-measurement-unit/static-product-measurement-unit-create.json

```diff
@@ -0,0 +1,6 @@
+{
+  "defaultPassword": "change123",
+  "rootUser": {
+    "username": "admin@spryker.com"
```

Comment:
Please use dynamic fixtures to have a backoffice user. We already have a helper for it. 
In case the default admin user is blocked, removed, or broken, all tests that rely on it will fail, which is wrong.